### PR TITLE
Toolbar

### PR DESCRIPTION
**Pull request by olearym**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,7 +54,11 @@
 
     <div id="EaselView" class="row">
             <div id="ToolbarView" class="col-md-1">
-                tools
+				<button type="button" class="btn btn-default toolbar-btn" id="select-btn"><img height="60px" src="assets/resources/button-images/grab.png"></button>
+				<button type="button" class="btn btn-default toolbar-btn" id="draw-btn"><img height="60px" src="assets/resources/button-images/pencil.png"></button>
+				<button type="button" class="btn btn-default toolbar-btn" id="erase-btn"><img height="60px" src="assets/resources/button-images/eraser.png"></button>
+				<button type="button" class="btn btn-default toolbar-btn" id="measure-btn"><img height="60px" src="assets/resources/button-images/ruler.png"></button>
```
Comment (mmou) [inline index.html:60]: can someone explain to me what 'ruler tool' does exactly?

Comment (olearym) [inline index.html:60]: like, apparently it's supposed to measure edges? I really don't think we need to worry about it now

File: index.html [hunk 2]
```diff
@@ -75,4 +79,4 @@
 
 </body>
```
Comment (mmou) [inline index.html:81]: you don't have to do this for this commit, but you will need the following frontend js code:

1) upon selecting a tool, your cursor should become that tool
2) upon selecting a tool, the corresponding tool button should still be "indented" (darker grey)

maybe this.toolbar.easel should have a function that sets currentTool, and also does these front end changes

File: assets/js/pad.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+function Pad() {
+    
+    this.sketch;
```
Comment (mmou) [inline assets/js/pad.js:3]: how do you set this Pad's sketch? should be a parameter for Pad, or there should be a function that allows you to access it. (might make more sense to change once ned's sketch code is pushed)

jk i just saw this.load. sorry

Comment (olearym) [inline assets/js/pad.js:3]: sketch is set with pad.load(sketch)

**Pull request merged**